docs/tutorials/onnx2tensorrt.md

```diff
@@ -50,6 +51,7 @@ Description of all arguments:
 - `--show`: Determines whether to show the outputs of the model. If not specified, it will be set to `False`.
 - `--verify`: Determines whether to verify the correctness of models between ONNXRuntime and TensorRT. If not specified, it will be set to `False`.
 - `--verbose`: Determines whether to print logging messages. It's useful for debugging. If not specified, it will be set to `False`.
+- `--skip-normalize`: Determins whether to skip normalization of the input image. If not specified, it will be set to `False`. This is useful when the normalization is already included in the ONNX graph.
```

Comment:
You could add note that this argument should be set to `True` only if `--normalize-in-graph` in `pytorch2onnx.py` is set to `True`.